Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -4617,37 +4632,82 @@ bool CStaticFunctionDefinitions::ResetWorldSounds()
     return true;
 }
 
-bool CStaticFunctionDefinitions::PlaySFX(CResource* pResource, eAudioLookupIndex containerIndex, int iBankIndex, int iAudioIndex, bool bLoop,
-                                         CClientSound*& outSound)
-{
+CClientSound* CStaticFunctionDefinitions::PlaySFX(
+    CResource* pResource,
+    eAudioLookupIndex containerIndex,
+    int iBankIndex,
+    int iAudioIndex,
+    bool bLoop
+) noexcept {
```

Comment:
Code formatting is still wrong. I am not going to mark all places in the review, it would be too much.

Reply:
Yeah I know. Im slowly transitioning to the right formatting, I want to push everything as one commit so it might take a while